### PR TITLE
[#169589538] Fix mjml raw template styling

### PR DESCRIPTION
**Pull request by AleDore**
A minimal fix to generated style on email template.

### CODE REVIEW

**Comment by gunzip** [comment]
can you please add a screenshot with before and after?

File: src/templates/html/default.ts [hunk 1]
```diff
@@ -33,7 +33,17 @@ export default function(
       }</style><style type="text/css">@media only screen and (max-width:480px) {
       table.mj-full-width-mobile { width: 100% !important; }
       td.mj-full-width-mobile { width: auto !important; }
-    }</style></head><body style="background-color:#FFFFFF;"><div style="background-color:#FFFFFF;"><!-- Header --><!-- <mj-section padding="10px 0">
+    }</style><style type="text/css">#outlook a { padding:0; }
```
Comment (gunzip) [inline src/templates/html/default.ts:36]: I guess we should use this instead:

https://documentation.mjml.io/#mj-style

File: templates/mjml/default.mjml [hunk 2]
```diff
@@ -1,12 +1,27 @@
 <mjml>
   <mj-head>
+    <mj-raw>
```
Comment (gunzip) [inline templates/mjml/default.mjml:3]: why not https://documentation.mjml.io/#mj-style ?

File: templates/mjml/default.mjml [hunk 3]
```diff
@@ -1,12 +1,27 @@
...
+      <style type="text/css">#outlook a { padding:0; }
+          .ReadMsgBody { width:100%; }
+          .ExternalClass { width:100%; }
+          .ExternalClass * { line-height:100%; }
```
Comment (gunzip) [inline templates/mjml/default.mjml:7]: can you add some comments on what are these classes and where do they come from?

File: templates/mjml/default.mjml [hunk 4]
```diff
@@ -1,12 +1,27 @@
...
+          body { margin:0;padding:0;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%; }
+          table, td { border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt; }
+          img { border:0;height:auto;line-height:100%; outline:none;text-decoration:none;-ms-interpolation-mode:bicubic; }
+          p { display:block;margin:13px 0; }</style><!--[if !mso]><!--><style type="text/css">@media only screen and (max-width:480px) {
+            @-ms-viewport { width:320px; }
+            @viewport { width:320px; }
```
Comment (gunzip) [inline templates/mjml/default.mjml:13]: these should be already in the generated html, can you please check?

**Review by gunzip — CHANGES_REQUESTED** [review]

**Review by gunzip — APPROVED** [review]

File: templates/mjml/default.mjml (hunk 4 above)
Comment (AleDore) [reply templates/mjml/default.mjml]: Yes it is present :)

**Pull request merged**